### ISSUE TITLE
Display AST at all steps of transformation (time travel)

### ISSUE THREAD

**Issue by ianjennings** [MLH-Fellowship/babel-sandbox#1]
From sandboxes created by [ianjennings](https://github.com/ianjennings): babel/sandboxes#17  

- [ ] show AST at all steps in the transformation (babel-time-travel)
    - archived repo: https://github.com/babel/babel-time-travel
    - PR to add to current repl https://github.com/babel/website/pull/1736

**Comment by ianjennings** [comment]
The original time travel is written in Vue.

If we look at the original babel pull, they convert it to react.

Personally I don't think the visualization they use represent the stages of deployment makes any sense, I think we should create our own.

Therefore, all we need is logic, which seems to exist here:
https://github.com/babel/website/pull/1736/files#diff-3964e83605d5471dc541a73f0e8efc83

The new exist criteria is:

- output an <HTML> list with each stage of timetravel into the UI

**Comment by wlawt** [comment]
### After some triage ...
Quick things to note from the PR tagged from babel/website:
- `.getValue()` seems to be the Array we want to iterate through
- To get that Array, `.addExitTransition(compiled)` appends the code from the result.js to the Array
- `.wrapPluginVisitorMethod()` _seems_ to be method we want, however, the inputs seem to be unclear given our environment being different from the PR in babel/website (referring to our `CompiledOutput.js`)

There also seems to be small ties to other files/variables, and these are some things we should consider if we need it:
- `state.timeTravel` -> bool
- `getTransitions` whose value is assigned to `state.TimeTravel`, which gets referenced [here](https://github.com/babel/website/pull/1736/files#diff-7eb9a407555c71b00fa2f51acce7226dR137-R139) 
- Do we add `wrapPluginVisitorMethod` to `babelConfig`? (see [this snippet](https://github.com/babel/website/pull/1736/files#diff-7eb9a407555c71b00fa2f51acce7226dL97-R131))

TL;DR the things that are blocking me:
1. Figuring out what to do above ^
2. What's the Array output that I'm supposed to expect? At the moment it looks like this with `env` plugin:
![image](https://user-images.githubusercontent.com/22751307/89351364-dd499880-d67f-11ea-99e2-846b4bf317ce.png)
